app/src/actions/counter.spec.ts

```diff
@@ -0,0 +1,64 @@
+import {mockStore} from '../mocks/mock-store';
+import {INCREMENT_COUNTER, DECREMENT_COUNTER} from './counter';
+import * as CounterActions from './counter';
+import {
+  describe,
+  it
+} from 'angular2/testing';
+
+describe('counter action creators', () => {
+  it('increment should create INCREMENT_COUNTER action', () => {
+    chai.expect(CounterActions.increment())
```

Comment:
If we are moving this from `Mocha` to `Jasmine`, should we still be using `Chai`?